config/examples/sim-lms.config

```diff
@@ -0,0 +1,50 @@
+# LMS/HSS signature example, based on sim.config example.
+#
+# LMS/HSS is a post-quantum, stateful, hash-based signature scheme.
+#
+# The acceptable parameter values are those in RFC8554:
+#   levels = {1..8}
+#   height = {5, 10, 15, 20, 25}
+#   winternitz = {1, 2, 4, 8}
+#
+# The number of available signatures is:
+#   N = 2 ** (levels * height)
+#
+# LMS/HSS Signature sizes are directly proportional to the levels parm,
+# and inversely proportional to Winternitz. They grow only modestly with
+# the height.
+#
+# Key generation time is strongly determined by the height of the first
+# level tree.
+#
+# Use the helper script
+#   tools/lms/lms_siglen
+# to calculate your signature length given the chosen levels, height,
+# Winternitz values.
+#
+
+ARCH=sim
+TARGET=sim
+SIGN?=LMS
+HASH?=SHA256
+LMS_LEVELS=2
+LMS_HEIGHT=5
+LMS_WINTERNITZ=8
+WOLFBOOT_SMALL_STACK=0
+SPI_FLASH=0
+DEBUG=0
+DELTA_UPDATES=0
+IMAGE_SIGNATURE_SIZE=2644
+IMAGE_HEADER_SIZE?=5288
+
+# it should be multiple of system page size
```

Comment:
"it" what is "it"?

Reply:
I updated it to be similar to the other sim*.config files.